src/amoc_api_scenario_handler.erl

```diff
@@ -10,112 +10,125 @@
          allowed_methods/2,
          content_types_provided/2,
          content_types_accepted/2,
-         process_request/2,
-         process_json/2]).
+         malformed_request/2,
+         resource_exists/2,
+         handle_get/2,
+         handle_patch/2]).
 
--record(state, {action}).
+
+-record(state, {resource, users}).
```

Comment:
I don't think that we need this state. We are going to simply retrieve the users and the scenario name in the callback for the appropriate HTTP method.